public/app/features/dashboard/save_modal.ts

```diff
@@ -14,15 +14,24 @@ const template = `
   </div>
 
   <form name="ctrl.saveForm" ng-submit="ctrl.save()" class="modal-content" novalidate>
-    <h6 class="text-center">Add a note to describe your changes</h6>
     <div class="p-t-2">
+      <div class="gf-form-group">
+		    <gf-form-switch class="gf-form"
```

Comment:
Maybe personal opinion, but in general would be nice if these only was visible when you actually have changed variables or time range and also show the difference original/current.